crates/stackable-versioned-macros/src/codegen/common/mod.rs

```diff
@@ -32,6 +32,26 @@ pub(crate) struct ContainerVersion {
 
     /// The ident of the container.
     pub(crate) ident: Ident,
+
+    /// Store an additional doc-comment lines for this version.
```

Comment:
suggestion: Reword sentence.

```suggestion
    /// Store additional doc-comment lines for this version.
```